src/main/psxsdk/libsnd/seqinit.c

```diff
@@ -1,10 +1,106 @@
 #include "common.h"
+#include "libsnd_i.h"
 
-s16 _SsInitSoundSeq(s16, s16, u32);
-
-INCLUDE_ASM("main/nonmatchings/psxsdk/libsnd/seqinit", _SsInitSoundSeq);
-
-extern u32 _snd_openflag;
+s16 _SsInitSoundSeq(s16 arg0, s16 vab_id, u8* addr) {
+    s32 temp_lo;
+    s32 tempo;
+    s32 delta_value;
+    s32 var_v1;
+    u32 temp_a1_2;
+    u32 temp_lo_2;
+    u32 temp_lo_3;
+    u32 temp_lo_4;
+    struct SeqStruct* score;
+    int new_var;
+    u32 new_var2;
+    u8 a;
+    u8 b;
+    u8 c;
+    u8 t;
+    u8 q;
+    var_v1 = 0;
+    score = &_ss_score[arg0][0];
+    score->unk4c = vab_id;
+    score->unk4a = 0;
+    score->unk13 = 0;
+    score->unk14 = 0;
+    score->unk29 = 0;
+    score->unk15 = 0;
+    score->unk16 = 0;
+    score->unk2a = 0;
+    score->channel = 0;
+    score->unk7c = 0;
+    score->unk80 = 0;
+    score->unk84 = 0;
+    score->unk72 = 0U;
+    score->unk48 = 0;
+    score->unk2b = 0;
+    score->delta_value = 0;
+    score->unk27 = 0;
+    score->unk28 = 0;
+    score->unk10 = 0;
+    score->unk11 = 0;
+    score->padA6 = 0x7F;
+    score->padaa = 0;
+    do {
+        score->programs[var_v1] = var_v1;
+        score->panpot[var_v1] = 0x40;
+        score->vol[var_v1] = 0x7f;
+        var_v1 += 1;
+    } while (var_v1 < 0x10);
```

Comment:
```suggestion
  for (var_v1 = 0; var_v1 < 0x10; var_v1++)
  {
    score->programs[var_v1] = var_v1;
    score->panpot[var_v1] = 0x40;
    score->vol[var_v1] = 0x7f;
  }
```
this can be rewritten as a for loop. Maybe we can also give a nicer name to `var_v1`